src/state/slices/opportunitiesSlice/selectors.test.ts

```diff
@@ -268,6 +271,7 @@ describe('opportunitiesSlice selectors', () => {
           tvl: '91283233211',
           type: DefiType.LiquidityPool,
           underlyingAssetIds: foxEthPair,
+          underlyingAssetRatios: ['0.005', '202.2'] as [string, string],
```

Comment:
mmmmm these should probably be in base units

Reply:
Definitely, that will avoid issues with precision later on 🐐
27f0799a3b2504ca15b7515ab4d65d0d645648aa